### DISCUSSION TITLE
Can a Service Bus scaler scale on a Dead-Letter queue?

### DISCUSSION THREAD

**Discussion by lyubomirhristovv** [kedacore/keda#4996] [Q&A / Need Help]
Hi,

I'd like to scale up containers based on messages count in DLQ, in order to process them.
So far, I've tried pointing to DLQ this way - "queueName": "test-queue/$DeadLetterQueue", but that was not successful.
Is there a way to scale on a Dead-Letter queue messages count?

**Comment by tomkerkhove** [comment]
Do you have any logs of the failed scenario?

**Reply by just-another-dev88** [reply]
this might help. 

![image](https://github.com/kedacore/keda/assets/81784594/c565c8e8-4fc7-41d2-9863-f6bdd136e31c)

**Reply by tomkerkhove** [reply]
It would be better to share the full logs though, please share these so we can investigate
